src/mixins/filing-mixin.ts

```diff
@@ -58,7 +58,7 @@ export default class FilingMixin extends Vue {
     priority: boolean | undefined,
     waiveFees: boolean | undefined
   ): void {
-    let myFilingData: Array<FilingDataIF> = this.filingData
+    let myFilingData: Array<FilingDataIF> = [...this.filingData]
```

Comment:
I noticed that, sometimes, the Fee Summary did not update itself even though the filing data was changed. I think this because we restructured filing data in the template in App.vue and it wasn't always reactive (I don't know why). THIS code change makes the new filing data a new object (and I removed the restructuring in App.vue) and now the Fee Summary always updates.